hedera-node/configuration/compose/bootstrap.properties

```diff
@@ -1,3 +1,3 @@
 bootstrap.throttleDefsJson.resource=throttles-dev.json
 accounts.blocklist.enabled=false
-accounts.blocklist.resource=
+accounts.blocklist.path=
```

Comment:
Do we need these overrides in all files under `configuration`. with empty values `accounts.blocklist.path=` ? If we don't mention anything here it should take default